NewForce-at-Mountwest/bangazon-workforce-management-the-lemony-slaps
# Sarah employee details

# Description
User can see details of each employee

## Related Ticket(s)
https://trello.com/c/UJXCCUCR/3-3-hr-should-be-able-to-view-an-employees-details

## Steps to Test

- Add and commit your branch
- Pull down branch "sarah-employeeDetails
- Check appsettings to make sure it is the correct database for your machine
- Run application with ctrl + F5
- Add /employees/details/2 onto URL 

## You should see:
- Derek Stapleton
- Dept Name
- Assigned Computer Make/Model
- Training Programs he is in
